source/rust_verify/src/rust_to_vir.rs

```diff
@@ -425,6 +425,11 @@ fn check_item<'tcx>(
             // type alias (like lines of the form `type X = ...;`
             // Nothing to do here - we can rely on Rust's type resolution to handle these
         }
+        ItemKind::GlobalAsm(..) =>
+        //if get_verifier_attrs(ctxt.tcx.hir().attrs(item.hir_id()))?.external =>
```

Comment:
Please remove this comment and add a TODO to add a crate-level attribute to enable `global_asm`.

Reply:
Fixed this comment.